runners-manager/vm_creation/openstack.py

```diff
@@ -0,0 +1,80 @@
+import os
+import time
+import datetime
+import keystoneauth1.session
+import keystoneclient.auth.identity.v3
+import neutronclient.v2_0.client
+from novaclient import client
+from pprint import PrettyPrinter
+
+from vm_creation.github_actions_api import link_download_runner
+
+pprinter = PrettyPrinter()
+
+keystone_endpoint = 'https://scality.cloud/keystone/v3'
+token = os.getenv("CLOUD_9_TOKEN")
+tenant_id = os.getenv("CLOUD_9_TENANT")
+
+region = 'Europe'
+session = keystoneauth1.session.Session(
+    auth=keystoneclient.auth.identity.v3.Token(
+        auth_url=keystone_endpoint,
+        token=token,
+        project_id=tenant_id)
+)
+nova_client = client.Client(version=2, session=session, region_name=region)
+neutron = neutronclient.v2_0.client.Client(session=session, region_name=region)
+
+
+def script_init_runner(name, token, labels, group):
+    installer = link_download_runner('scalanga-devl')
```

Comment:
something to keep in mind, but we need a variable here before I can actually set something up deployed